### ISSUE TITLE
Use Bazel in deployment scripts [phab:D278]

### ISSUE THREAD

**Issue by kythe** [google/kythe#497]
_Created by **schroederc** at 2015-05-22 22:53:47_: 

NOTE: The xrefs deployment is broken again, but it's not because of these changes.  I'll fix it in another change.

**Comment by kythe** [comment]
_**zarko** wrote at 2015-05-22 23:25:57_:

I'm having some trouble with deploy.sh. I have to run it twice, since the first time it complains about extra_actions:

```
DEV=1 ./kythe/release/appengine/xrefs/deploy.sh
Extracting Kythe compilations to /tmp/tmp.wCdHoc53GS.kythe_compilations
find: `bazel-out/local_linux-fastbuild/extra_actions': No such file or directory
```

Then it will run for a bit and print:
```
Skipping extraction
write_entries: error while loading shared libraries: libleveldb.so.1: cannot open shared object file: No such file or directory
Indexing /compilations/c++/kythe_cxx_extractor_extra_action_third_party_grpc_eb3e7db8e0a72d0a5c02579609ed71b4.c++.kindex
2015/05/22 23:22:04 write /dev/stdout: broken pipe
141 analysis failures
```
I did do a `bazel build //kythe/release:docker` right before calling `deploy.sh`.

**Comment by kythe** [comment]
_**schroederc** wrote at 2015-05-26 16:31:45_:

Fix xrefs deploy

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2015-05-26 17:55:08_

**Comment by kythe** [comment]
_**schroederc** closed via commit f00ab3adb48953adf5606f5d7a9db8ac23d75d31 at 2015-05-26 17:56:01_
